rmgpy/statmech/conformerTest.py

```diff
@@ -34,12 +34,12 @@
 
 from __future__ import division
 
-import unittest
 import numpy as np
+import unittest
```

Comment:
`unittest` is actually part of the standard library, so it should go in the first section, before numpy.